gui/DataColorOptions.cxx

```diff
@@ -113,6 +115,22 @@ vtkScalarsToColors* DataColorOptions::scalarsToColors() const
 }
 
 //-----------------------------------------------------------------------------
+double DataColorOptions::getMinValue()
```

Comment:
Aaaaaand... API should (almost) never be abbreviated, so actually it should be `minimumValue`, or probably just `minimum`.